cluster-operator/src/main/java/io/strimzi/operator/cluster/model/EntityOperator.java

```diff
@@ -319,4 +324,31 @@ public Role generateRole(String ownerNamespace, String namespace) {
 
         return role;
     }
+
+    /**
+     * Generates the NetworkPolicies relevant for Entity Operator
+     *
+     * @param operatorNamespace                             Namespace where the Strimzi Cluster Operator runs. Null if not configured.
+     * @param operatorNamespaceLabels                       Labels of the namespace where the Strimzi Cluster Operator runs. Null if not configured.
+     *
+     * @return The network policy.
+     */
+    public NetworkPolicy generateNetworkPolicy(String operatorNamespace, Labels operatorNamespaceLabels) {
+        // List of network policy rules for all ports
+        List<NetworkPolicyIngressRule> rules = new ArrayList<>();
+
+        // Everyone can access metrics
+        if (metrics.isEnabled()) {
+            rules.add(NetworkPolicyUtils.createIngressRule(MetricsModel.METRICS_PORT, List.of()));
+        }
```

Comment:
The metrics are always enabled. They are also on two ports:
* If Topic Operator is enabled, its metrics will be on 8080
* If User Operator is enabled, its metrics will be on 8081

Reply:
rules.add(NetworkPolicyUtils.createIngressRule(EntityTopicOperator.HEALTHCHECK_PORT, List.of()));

        // For User Operator
        rules.add(NetworkPolicyUtils.createIngressRule(EntityUserOperator.HEALTHCHECK_PORT, List.of()));
 Added ports like this..
 Should I add If statement as  UserOperator or TopicOperator can be null...

Reply:
Yes, ideally that should be checked.